packages/salesforcedx-vscode-replay-debugger/package.nls.json

```diff
@@ -4,6 +4,7 @@
   "launch_snippet_description_text":
     "A new configuration for launching Apex Replay Debugger",
   "launch_snippet_name": "Launch Apex Replay Debugger",
+  "launch_from_log_file": "Launch Apex Replay Debugger with current file",
```

Comment:
@ruthemmanuelle In the .GIF, this is the label for the menu item.

Reply:
The other menu items use title-case capitalization, so I'd go with "Launch Apex Replay Debugger with Current File"—or, if you don't think it's too long, "Launch Apex Replay Debugger with Current Log File".

Reply:
Okay, I'm also going to prefix it with `SFDX:` because this command shows up in the command palette as well.